pages/quickstart/setting-up-a-local-environment-with-dive-cli.mdx

```diff
@@ -0,0 +1,165 @@
+
+- <b>--nid:</b> Assign the network ID (NID) of your ICON node using the --nid flag. The NID uniquely identifies your ICON network.
+
+- <b>--nodeEndpoint:</b> Provide the endpoint address of your already running ICON node using the --nodeEndpoint flag. This endpoint allows DIVE CLI to interact with the existing ICON node.
+
+- <b>--serviceName:</b> Assign a custom container name for the decentralized ICON node using the --serviceName flag. This name will be used to identify the container during the decentralization process.
+
+After executing the command, DIVE CLI will initiate the decentralization process for your existing ICON node. Please wait for the process to complete, as this may take some time depending on the size of your blockchain and network.
+
+### Ethereum
+
+To start Eth Node, run the following command
+
+```sh
+dive chain eth
+```
+
+After running the command, DIVE CLI will automatically start the ETH node and handle the necessary initialization processes. Please wait for the ETH node to fully initialize, which may take a few moments.
+
+<Callout type="info" emoji="💡">
+  <b>TIP:</b> DIVE cli also supports hardhat. You can start an hardhat Node by running
+  the command ``` dive chain hardhat ```
+</Callout>
+
+## Setting up Cross chain connection
+
+To start both the ICON and Ethereum chains and establish a BTP bridge between them, run the following command:
+
+```sh
+dive bridge btp --chainA icon --chainB eth
+```
+
+<Callout type="info" emoji="💡">
+  
+    <b>TIP:</b> By default, the bridge flag is set to false which uses BMV for BTPBlock
+    . If you need an BMV for Bridge , add the `--bridge` flag to the command.
+
+</Callout>
+
+The above command automates several essential tasks to configure and initialise the local environment:
+
+- <b>Spinning up ICON and Ethereum Nodes:</b> DIVE automatically starts and configures the ICON and Ethereum nodes required for your local environment.
+
+- <b>Installing Required npm Dependencies:</b> DIVE takes care of installing all the necessary npm dependencies for smooth operation.
+
+- <b>Enabling BTP Block on ICON Local Network:</b> By registering the node as a PRep and decentralizing the network, DIVE enables BTP block functionality on the ICON local network.
+
+- <b>Deploying Contracts for BTP:</b> DIVE automatically builds and deploys the required smart contracts (BMC, BMV, xCall & sample dApp) for BTP functionality on both the ICON and Ethereum chains.
+
+- <b>Starting the Relay:</b> The relay image is built and launched, actively monitoring both the ICON and Ethereum chains to facilitate seamless message transfer between them.
+
+<Callout type="info" emoji="💡">
+  <b>TIP:</b> Also you can setup a BTP bridge between (Icon-Icon) by running followng
+    command `dive bridge btp --chainA icon --chainB icon`
+
+</Callout>
+
+Once the process completes, your local environment with the ICON and Ethereum chains connected via the BTP bridge will be ready for testing and development.
```

Comment:
Add an explanation of the outputs of the command. This would include ALL of the docker containers and dive.json. Currently, it isn't clear how dive.json is used and how to go from local development to testnet.